arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-custom-gas-token-chain.md

```diff
@@ -0,0 +1,137 @@
+---
+title: 'How to Deploy a Custom Gas Token Chain Using the Orbit SDK'
+sidebar_label: 'Custom Gas Token Orbit Deployment'
+description: 'How to deploy a custom gas token chain using the Orbit SDK'
+author: GreatSoshiant
+sme: GreatSoshiant
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 3
+---
+
+:::caution UNDER CONSTRUCTION
+
+This document is under construction and may change significantly as we incorporate [style guidance](/for-devs/contribute#document-type-conventions) and feedback from readers. Feel free to request specific clarifications by clicking the `Request an update` button at the top of this document.
+
+:::
+
+:::info
+
+See the ["create a rollup custom fee token" example](https://github.com/OffchainLabs/arbitrum-orbit-sdk/blob/main/examples/create-rollup-custom-fee-token/index.ts) in the Orbit SDK repository for additional guidance.
+
+:::
+
+
+Deploying a Custom Gas Token Orbit chain introduces a unique aspect to the standard Orbit chain setup: the ability to pay transaction fees using a specific `ERC-20` token instead of `ETH`. While the setup process largely mirrors that of a standard <a data-quicklook-from="arbitrum-rollup-chain">Rollup Orbit chain</a>, as detailed in the [introduction](../orbit-sdk-introduction.md), there are key differences to account for when configuring a Custom Gas Token Orbit chain.
+
+:::important
+
+Custom gas tokens are only supported on Orbit AnyTrust chains, currently. This feature is **not available yet** on Rollup Orbit chains.
+
+:::
+
+## Key Differences for Custom Gas Token Orbit Chain Deployment
+
+### 1. Fee Token Specification 
```

Comment:
```suggestion
### 1. Custom gas token specification 
```
Stepping into Product's realm here, but I believe this is the concept that's being marketed (also, it's referenced as "custom gas token" in the rest of the doc)